frontend/src/metabase/embedding-sdk/theme/dynamic-css-vars-config.ts

```diff
@@ -0,0 +1,21 @@
+import type { DynamicCssVarConfig } from "../types/private/css-variables";
+
+/**
+ * These CSS variables are dynamically generated based on the theme.
+ */
+export const DYNAMIC_CSS_VARIABLES: DynamicCssVarConfig = {
+  "--mb-color-notebook-step-bg": {
+    source: "bg-white",
+    light: { darken: 0.05 },
+    dark: { lighten: 0.5 },
+  },
+  "--mb-color-notebook-step-bg-hover": {
+    source: "bg-white",
+    light: { darken: 0.1 },
+    dark: { lighten: 0.4 },
+  },
+  "--mb-color-background-hover": {
+    source: { dark: "bg-white" },
```

Comment:
why can't we use `source: 'bg-white'` here like others? We don't override light theme here anyway.

Reply:
@WiNloSt This is because it would break the light theme. I tried to match `alpha` and `lighten` to the current value in the main Metabase app for the light theme, but it would not match the current `--mb-color-background-hover` and looks very wrong.

I decided to only override in the dark palette for now until I can figure out how to match that color. My best guess it is either to do with input color spaces (RGB vs sRGB), or differences between how `lighten/darken` and `color-mix` mixes.